lists_04/find_number.py

```diff
@@ -5,4 +5,11 @@
 
 
 def find_number(x_arr, x):
-    pass
+    if x not in x_arr:
```

Comment:
Please do not use list methods. implement your own way of finding a  number from a list